src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/IndividualBeneficialOwnerCessation.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import uk.gov.companieshouse.overseasentitiesapi.service.changelist.PersonName;
+
+public class IndividualBeneficialOwnerCessation extends BeneficialOwnerCessation {
+
+    @JsonProperty("personName")
+    private PersonName personName;
+
+    @JsonProperty("birthDate")
+    private LocalDate birthDate;
+
+    public IndividualBeneficialOwnerCessation(
```

Comment:
I think we need to set the change name to 'ceasePSCAppointment' for Legal, Corporate and Individual BOs. By default it's 'pscCessation' which you already have set

Reply:
just change the default?